I am trying to bump the MinIO version because the current version has a spurious segfault that causes the tests to fail

```
Formatting 1st pool, 1 set(s), 1 drives per set.
WARNING: Host local has more than 0 drives of set. A host failure will result in data becoming unavailable.
WARNING: Detected default credentials 'minioadmin:minioadmin', we recommend that you change these values with 'MINIO_ROOT_USER' and 'MINIO_ROOT_PASSWORD' environment variables
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x70 pc=0x1f1acfa]

goroutine 53 [running]:
github.com/minio/minio/cmd.(*ReplicationStats).getAllCachedLatest(0x40f4df?)
	/workspace/srcdir/minio/cmd/bucket-replication-stats.go:249 +0x3a
github.com/minio/minio/cmd.(*ReplicationPool).saveStats(0x0, {0x5200700, 0xc0000580e8})
	/workspace/srcdir/minio/cmd/bucket-replication.go:2798 +0x4d
github.com/minio/minio/cmd.(*ReplicationPool).SaveState(0x0, {0x5200700, 0xc0000580e8})
	/workspace/srcdir/minio/cmd/bucket-replication.go:2823 +0x79
github.com/minio/minio/cmd.handleSignals()
	/workspace/srcdir/minio/cmd/signals.go:81 +0xfe
created by github.com/minio/minio/cmd.serverMain
	/workspace/srcdir/minio/cmd/server-main.go:424 +0xca
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0x1f4693c]

goroutine 39 [running]:
github.com/minio/minio/cmd.(*ReplicationPool).SaveState.func1()
	/workspace/srcdir/minio/cmd/bucket-replication.go:2818 +0x1c
created by github.com/minio/minio/cmd.(*ReplicationPool).SaveState
	/workspace/srcdir/minio/cmd/bucket-replication.go:2816 +0x65
```

--- a/Project.toml
+++ b/Project.toml
@@ -31,7 +31,7 @@ SHA = "0, 1, 2"
 URIs = "1"
 azurite_jll = "3"
 julia = "1.6"
-minio_jll = "1"
+minio_jll = "2"
 
 [extras]
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"

